index.html

```diff
@@ -27,9 +27,7 @@
           <a href="#about">About</a>
         </li>
         <li>
```

Comment:
- [ ] You did a good job on the functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. Please fix the below-mentioned issues.
  - When the mobile menu is open, Please make sure the title and hamburger icon is not visible. A close(X) button should be visible on the menu to close it.
  - Please use the correct color for mobile menu options. `#FFF5E1`
  - Please make sure the mobile menu contains 100% of the width and height of the screen. It will improve the UI of your portfolio.

### Your design

![image](https://user-images.githubusercontent.com/83803414/175037527-bcc437e5-4e63-4f0b-9999-b5495216ccca.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/175037598-777c226a-677e-4a5c-84fe-0f6dec7c6976.png)